.travis.yml

```diff
@@ -6,13 +6,18 @@ python:
 
 sudo: "required"
 
-before_install:
-  - "pip install virtualenv"
-  - "virtualenv venv"
-  - "source venv/bin/activate"
+env:
+  matrix:
+  - TOXENV=foxpuppet GUI=1
```

Comment:
No need for a matrix if we only have one entry. I've suggested having py27 and py35 as test environments, so let's have those two items in the matrix.